mlos_bench/mlos_bench/tunables/tunable_groups.py

```diff
@@ -27,12 +27,14 @@ def __init__(self, config: dict = None):
         config : dict
             Python dict of serialized representation of the covariant tunable groups.
         """
-        self._index = {}  # Index (Tunable id -> CovariantTunableGroup)
-        self._tunable_groups = {}
-        for (name, group_config) in (config or {}).items():
+        if config is None:
+            config = {}
+        self._index: Dict[str, CovariantTunableGroup] = {}  # Index (Tunable id -> CovariantTunableGroup)
+        self._tunable_groups: Dict[str, CovariantTunableGroup] = {}
+        for (name, group_config) in (config).items():
```

Comment:
```suggestion
        for (name, group_config) in config.items():
```
😄

Reply:
Huh, don't remember doing that, but there's a lot of changes here 😅

Reply:
I blame copilot :P